styles/custom.scss

```diff
@@ -3,6 +3,30 @@ $theme-colors: (
   "secondary": #00bfa5,
 );
 
+$link-color: #0d3e79;
+$hr-color: #dee2e6;
+$hr-opacity: 1;
+$link-decoration: none;
+
+body {
+  font-family: -apple-system,
+    // Windows
+    "Segoe UI",
+    // Android
+    Roboto,
+    // Basic web fallback
+    "Helvetica Neue",
+    Arial,
+    // Linux
+    "Noto Sans",
+    "Liberation Sans",
+    // Sans serif fallback
+    sans-serif,
+    // Emoji fonts
+    "Apple Color Emoji",
+    "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji" !important;
```

Comment:
this part i'm not sure about - the upgrade changed the font of body across the website so i added this to make the `font-family` the same as it was before. is this okay?

Reply:
I looked at this locally with and without this override so that I could try to answer this, and I'm seeing the same font either way. Could be an OS thing?